src/main/java/com/ipiecoles/java/java220/Employe.java

```diff
@@ -1,7 +1,148 @@
 package com.ipiecoles.java.java220;
 
+import org.joda.time.LocalDate;
+
+import java.util.Objects;
+
 /**
  * Created by pjvilloud on 21/09/17.
  */
-public class Employe {
+public abstract class Employe {
+    private String nom = new String();
+    private String prenom = new String();
+    private String matricule = new String();
+    private LocalDate dateEmbauche;
+    private Double salaire;
+    private Boolean tempsPartiel;
+    private String sexe;
+
+    public Employe(){
+    }
+
+    public Employe(String nom, String prenom, String matricule, LocalDate dateEmbauche, Double salaire){
+        this.nom = nom;
+        this.prenom = prenom;
+        this.matricule = matricule;
+        this.dateEmbauche = dateEmbauche;
+        this.salaire = salaire;
+    }
+
+    public Employe(String nom, String prenom, String matricule, LocalDate dateEmbauche, Double salaire, Boolean tempsPartiel, String sexe){
+        this.nom = nom;
+        this.prenom = prenom;
+        this.matricule = matricule;
+        this.dateEmbauche = dateEmbauche;
+        this.salaire = salaire;
```

Comment:
Utilise le constructeur déjà existant au lieu de dupliquer son contenu
```suggestion
        this(nom, prenom, matricule, dateEmbauche, salaire);
```